server/views/pages/overview/partials/supportNeedsTabContents.test.ts

```diff
@@ -95,3 +76,55 @@ describe('Support Needs tab view', () => {
     expect($('h2').text()).toEqual('Sorry, the Curious service is currently unavailable.')
   })
 })
+
+declare module 'cheerio' {
+  interface Cheerio<T> {
+    heading(this: Cheerio<T>): Cheerio<T>
+    rapidAssessmentDate(this: Cheerio<T>): Cheerio<T>
+    inDepthAssessmentDate(this: Cheerio<T>): Cheerio<T>
+    primaryLddAndHealthNeeds(this: Cheerio<T>): Cheerio<T>
+    additionalLddAndHealthNeeds(this: Cheerio<T>): Cheerio<T>
+    selfDeclaredNeurodiversityDiagnosis(this: Cheerio<T>): Cheerio<T>
+    assessedNeurodiversityDiagnosis(this: Cheerio<T>): Cheerio<T>
+    neurodiversitySupportNeeds(this: Cheerio<T>): Cheerio<T>
+  }
+}
+
+const setupCheerioExtensionFunctions = ($: CheerioAPI) => {
+  // eslint-disable-next-line no-param-reassign
+  $.prototype.heading = function heading(): Cheerio<never> {
+    return this.find('h3').text()
+  }
+  // eslint-disable-next-line no-param-reassign
+  $.prototype.rapidAssessmentDate = function rapidAssessmentDate(): Cheerio<never> {
```

Comment:
I can't really explain the return type of `Cheerio<never>` other than to say `never` is possibly a bit like `any`, but typescript and/or the linter moan about using `Cheerio<any>` and tell you to use `Cheerio<never>` instead !